### PR TITLE
Email script for comms on gdrive sync issue

### PR DESCRIPTION
**Pull request by philipperolet**
First test by sending it only to dust
Then add the correct resolution time, and all other workspace ids

### CODE REVIEW

File: front/mailing/20231108-gdrive-issue.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+export const sendAPIUserEmail = async ({
+  user_email,
+}: {
+  user_email: string;
+}) => {
+  const msg = {
+    to: user_email,
+    from: "spolu@dust.tt",
+    subject:
+      "[Dust] Resolved: Google Drive most recent documents not available for retrieval",
```
Comment (spolu) [inline front/mailing/20231108-gdrive-issue.ts:17]: I would say "Google Drive out of sync for 3h" ? please check final copy with @gabhubert :+1:

File: front/mailing/20231108-gdrive-issue.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    html: `<p>Hi there—</p>
+    <p>I'm Stan, a cofounder of Dust.</p>
+    <p>Thanks a lot for trying us out. Between 10:18 UTC and XXX UTC google drive syncing was interrupted due to a problem on our end. During that time, assistants performing retrieval did not have access to your latest Drive documents.</p>
```
Comment (spolu) [inline front/mailing/20231108-gdrive-issue.ts:20]: They're paying customers so they're not trying us out. Can be removed

Comment (spolu) [inline front/mailing/20231108-gdrive-issue.ts:20]: ... but everything else worked normally ?

File: front/mailing/20231108-gdrive-issue.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <p>Sincere apologies for that, it's not the experience we wanted you to have when giving us a spin. This is now resolved and the google drive connection is fully synced.</p>
```
Comment (spolu) [inline front/mailing/20231108-gdrive-issue.ts:21]: giving us a spin ===> to have as a paying user of Dust.

**Comment by philipperolet** [comment]
Ack to all coms, will send for dust and @gabhubert can review the email directly

**Review by spolu — DISMISSED** [review]
LGTM Left some wording comments that you can iron out with @gabhubert IRL

File: front/mailing/20231108-gdrive-issue.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+async function main() {
+  const [rows] =
+    await front_sequelize.query(`SELECT "data_sources"."connectorId", "workspaces"."sId", "workspaces"."name", "users"."email" user_email
+  FROM "data_sources"
+  JOIN "workspaces" ON "data_sources"."workspaceId" = "workspaces"."id"
+  JOIN "memberships" ON "workspaces"."id" = "memberships"."workspaceId"
+  JOIN "users" ON "memberships"."userId" = "users"."id"
+  WHERE "data_sources"."connectorId" IN ('24')
```
Comment (spolu) [inline front/mailing/20231108-gdrive-issue.ts:39]: Update query?

**Comment by spolu** [comment]
I think we can close (was superseded by the OAI incident)

**Pull request closed without merging**